src/data/faq_de.json

```diff
@@ -54,6 +54,14 @@
                 "title": "Häufige Probleme",
                 "id": "most_asked",
                 "accordion": [
+                    {
+                        "title": "[Apple/iOS]: Warum sehe ich einen Bildschrim der mir sagt das ich mein Betriebssystem aktualisieren muss?",
+                        "anchor": "update_iOS",
+                        "active": true,
+                        "textblock": [
+                            "Dieser Bildschirm wird auf iPhones mit einer iOS-Version zwischen iOS 13.5 und (einschließlich) iOS 13.6.1 angezeigt. Auf diesen iOS-Versionen ist die verwendete Version 2 des Exposure Notification Frameworks nicht verfügbar, deshalb muss erst auf iOS 13.7 oder neuer aktualisiert werden, um die Corona-Warn-App weiterhin verwenden zu können. Weitere Informationen finden Sie in <a href='../faq/#ios135' target='_blank' rel='noopener noreferrer'>diesem</a> FAQ Eintrag."

```

Comment:
I think "benötigte Version" is better than "verwendete Version" which also fits what I suggested for the English text.